buildpipeline/Core-Setup-PortableLinux-x64.json

```diff
@@ -97,8 +97,180 @@
     },
     {
       "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Copy built Portable binaries to staging directory",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "5bfb729a-a7c8-4a78-a7c3-8d717bb7c13c",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "SourceFolder": "$(Build.SourcesDirectory)/artifacts/linux-x64/intermediate/sharedFrameworkPublish",
```

Comment:
cc @gkhanna79  
if I don't call git clean step and artifacts are left behind from the previous build , I have noticed that coreclr fails to initialize in the prepare step. @gkhanna79  has more insights on why this happens.  The same approach of calling  git clean is done for portable Linux workflow https://github.com/dotnet/core-setup/blob/master/buildpipeline/Core-Setup-PortableLinux-x64.json#L92 here.